### ISSUE TITLE
Document the random behavior of choosing which files are being read?

### ISSUE THREAD

**Issue by kwinz** [ncw/stressdisk#16]
Right now the documentation states:
`It then randomly chooses a pair of these and reads them back checking they are the same.`

So by reading this I would assume that due to random chance it would be possible that even if I let this run for multiple days one file was already read e.g. 4 times, while another file hasn't been read ever. In practice by looking at the implementation it seems to me that not just `stressdisk cycle` but also `stressdisk run` is implemented in rounds. And during each round each file is read exactly twice before progressing to the next round. (Ignoring the unlucky case where one file would be selected to be compared to itself then that check would probably be skipped).
Sorry if I got anything wrong here, I just had a brief glance at the code.

Do we want to document this behaviour? I think sometimes it's useful to make sure that every file has been read at least once if you want to test some hardware. I know stressdisk's main purpose is seems to be to create a lot of seek stress to test the drive this way, but ensuring every file was tried at least once as soon as it prints "Starting round 2" would be a nice additional guarantee.

Or is this implementation detail undocumented on purpose? E.g. so the implementation can change in the future.

**Comment by ncw** [comment]
> And during each round each file is read exactly twice before progressing to the next round. (Ignoring the unlucky case where one file would be selected to be compared to itself then that check would probably be skipped).

Yes that is how it works, each file is read twice per round on a random schedule. Also it is never compared to itself.

So the docs could be more accurate! Do you want to suggest better wording?
